src/react/site/home/index.tsx

```diff
@@ -56,48 +61,62 @@ function Home({ onLoad }: { onLoad: () => void }) {
               >
-                <div className="min-h-12 bg-surface w-full p-2">
-                  <div className="flex items-start gap-2">
-                    <a
-                      className="hover:scale-125 duration-200 transition"
-                      href={`/foro/${thread.forumSlug}`}
-                    >
-                      <i
-                        className={clsx(
-                          "fid",
-                          getIconClassBySlug(thread.forumSlug)
-                        )}
-                      />
-                    </a>
-                    <a
-                      href={thread.url}
-                      title={thread.title}
-                      className="text-sm font-medium line-clamp-2 text-primary min-h-[40px]  hover:underline"
-                    >
-                      {thread.title}
-                    </a>
-                  </div>
-                  <div className="text-gray-400 dark:text-gray-600 text-right mt-2">
-                    {thread.createdAt}
-                  </div>
-                </div>
+                <div className="min-h-12 bg-surface w-full p-2"></div>
               </div>
-            );
-          })}
+            ))
+          : lastNews
+              ?.filter((_, i) => i < MAX_NEWS)
+              .map((thread) => {
+                return (
+                  <div
+                    key={thread.url}
+                    className="flex flex-col bg-opacity-90 justify-end h-44 w-full rounded shadow"
+                    style={{
+                      backgroundImage: `url(${thread.thumbnail})`,
+                      backgroundSize: "cover",
+                      backgroundPosition: "center",
+                      backgroundRepeat: "no-repeat",
+                    }}
+                  >
+                    <div className="min-h-12 bg-surface w-full p-2">
+                      <div className="flex items-start gap-2">
+                        <a
+                          className="hover:scale-125 duration-200 transition"
+                          href={`/foro/${thread.forumSlug}`}
+                        >
+                          <i
+                            className={clsx(
+                              "fid",
+                              getIconClassBySlug(thread.forumSlug)
+                            )}
+                          />
+                        </a>
+                        <a
+                          href={thread.url}
+                          title={thread.title}
+                          className="text-sm font-medium line-clamp-2 text-primary min-h-[40px]  hover:underline"
+                        >
+                          {thread.title}
+                        </a>
+                      </div>
+                      <div className="text-gray-400 dark:text-gray-600 text-right mt-2">
+                        {thread.createdAt}
+                      </div>
+                    </div>
+                  </div>
```

Comment:
Next step IMO would be encapsulating this in a `News` component.

Reply:
Yes, feel free to do so if you feel like it.

Said that, the news presentation is totally open to debate, I did just hack something quick (Same for threads, if there are any changes to the visuals I'm open to it).

Reply:
(And with that also the NewsSkeleton so we shave code from this page)